app/frontend/src/features/communities/CommunityInfoPage.test.tsx

```diff
@@ -0,0 +1,192 @@
+
+  describe("when there are more than one page of moderators", () => {
+    it("should show a 'See all moderators' button which opens a dialog when clicked", async () => {
+      listAdminsMock.mockResolvedValue({
+        adminUserIdsList: [2, 3],
+        nextPageToken: "3",
+      });
+      render(<CommunityInfoPage community={community} />, { wrapper });
+      await waitForElementToBeRemoved(screen.getByRole("progressbar"));
+
+      userEvent.click(screen.getByRole("button", { name: SEE_ALL_MODERATORS }));
+      const adminDialog = within(await screen.findByRole("presentation"));
+
+      expect(
+        adminDialog.getByRole("heading", { name: COMMUNITY_MODERATORS })
+      ).toBeVisible();
+      assertAdminsShown(adminDialog);
+      expect(
+        adminDialog.getByRole("button", { name: LOAD_MORE_MODERATORS })
+      ).toBeVisible();
+    });
+  });
+
+  describe("when the moderators dialog is opened", () => {
+    beforeEach(async () => {
+      listAdminsMock.mockResolvedValue({
+        adminUserIdsList: [2, 3],
+        nextPageToken: "3",
+      });
+      render(<CommunityInfoPage community={community} />, { wrapper });
+      await waitForElementToBeRemoved(screen.getByRole("progressbar"));
+
+      userEvent.click(screen.getByRole("button", { name: SEE_ALL_MODERATORS }));
+      await screen.findByRole("presentation");
+    });
+
+    it("loads more moderators when the button is clicked", async () => {
+      listAdminsMock.mockResolvedValue({
+        adminUserIdsList: [4],
+        nextPageToken: "",
+      });
+
+      userEvent.click(
+        screen.getByRole("button", { name: LOAD_MORE_MODERATORS })
+      );
+      await waitForElementToBeRemoved(screen.getByRole("progressbar"));
+
+      const adminDialog = within(await screen.findByRole("presentation"));
+      assertAdminsShown(adminDialog);
+      expect(
+        adminDialog.getByRole("link", {
+          name: getProfileLinkA11yLabel(thirdAdmin.name),
+        })
+      ).toBeVisible();
+      expect(adminDialog.getByText(thirdAdmin.name)).toBeVisible();
+
+      // Check it doesn't affect the underlying page
+      userEvent.click(document.querySelector(".MuiBackdrop-root")!);
+      await waitForElementToBeRemoved(screen.getByRole("presentation"));
+      expect(
+        adminDialog.queryByRole("link", {
+          name: getProfileLinkA11yLabel(thirdAdmin.name),
+        })
+      ).not.toBeInTheDocument();
+      expect(adminDialog.queryByRole(thirdAdmin.name)).not.toBeInTheDocument();
+    });
+
+    it("closes the dialog by clicking the backdrop", async () => {
+      // Close dialog by clicking the backdrop
+      userEvent.click(document.querySelector(".MuiBackdrop-root")!);
+      await waitForElementToBeRemoved(screen.getByRole("presentation"));
+
+      expect(
+        screen.queryByRole("button", { name: LOAD_MORE_MODERATORS })
+      ).not.toBeInTheDocument();
+    });
+
+    it("closes the dialog by pressing the escape key", async () => {
```

Comment:
Isn't this testing mui?

Reply:
I thought about this, but in some places like the bug report dialog we explicitly disable some things like no clicking background for closing, so I guess this is checking this hasn't been disabled by accident